apps/js-machine-front/src/app/scenes/about/styles/tileContent.css

```diff
@@ -20,7 +20,7 @@
     margin-right: 22px;
     margin-top: 60px;
     overflow-y: scroll;
-    height: 55%;
+    height: calc(71vh - 60px);;
```

Comment:
Use calc and vh bad practice. Please change your solution

Reply:
Agree